pkg/controller/osm/ingress_backend_reconciler.go

```diff
@@ -136,9 +137,11 @@ func (i *IngressBackendReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			return result, client.IgnoreNotFound(err)
 		}
 
-		logger.Info("deleting IngressBackend")
```

Comment:
don't remove the log

Reply:
The log was just moved to the block where it's getting checked to see if we own it. Should I let it hit this even if it's something we don't own?